desktop/apps/consignments2/client/reducers.js

```diff
@@ -86,8 +68,8 @@ const initialState = {
   progressBars: {},
   resetPasswordSuccess: false,
   skipPhotoSubmission: false,
-  steps: sd && sd.CURRENT_USER ? last(stepsMapping, 3) : stepsMapping,
-  submission: {},
+  steps: [],
+  submission: sd.SUBMISSION || { signature: true, authenticity_certificate: true },
```

Comment:
I don't think this should be a problem-- it's mostly a way to populate the initial state of the form, but looking back it seems a little odd to do this here so I'll stick it in the form container component.